### DISCUSSION TITLE
Feature: Webhook event for tracking status changes of a payout

### DISCUSSION THREAD

**Discussion by ilearnio** [btcpayserver/btcpayserver#4932] [Ideas / Feature Requests]
We have a goal to implement the withdrawal (from BTCPay) system in our web app so that our customers can deposit and then withdraw their BTC. For deposits we use webhooks to track the deposit status updates but for withdrawal there are no webhooks events we could leverage. It would be nice to track status changes of payouts, because for now, as a temporary workaround we have to fetch payouts from BTCPay using cron job, but that's may slow down our app backend significantly if there will be hundreds of requests to BTCPay made (for each payout) when our cron job is running.

**Comment by pavlenex** [comment]
I think this one was introduced in 1.12. can you check @ilearnio ?
